python/fichas/listas.py

```diff
@@ -9,7 +9,7 @@
 """
 
 
-def enum_from_to(start: int, end: int) -> list[int]:
+def enum_from_to(start: int, end: int) -> list:
```

Comment:
```suggestion
def enum_from_to(start: int, end: int) -> list[int]:
```

I think we should keep the type of the list values.